### PR TITLE
Add service.json file

### PR DESCRIPTION
**Pull request by alexhumphreys**

### CODE REVIEW

File: deploy/service.json [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+    "PACKAGES_VERSION_FORMAT": ".+",
+    "SERVICE_NAME": "${CORE_JOB_NAME}",
+    "SERVICE_SCOPE": "internal",
+    "SERVICE_TYPE": "http",
+    "SERVICE_IMAGE": "advancedtelematic/sota-core:${DOCKER_TAG}",
+    "SERVICE_HEALTH_CHECK": "curl --show-error --silent %<host>s:%<port>s/health",
+    "TUF_REPOSERVER_HOST": "${TUF_REPOSERVER_HOST}",
+    "TUF_REPOSERVER_PORT": "${TUF_REPOSERVER_PORT}",
+    "USER_PROFILE_HOST": "${USER_PROFILE_HOST}",
+    "USER_PROFILE_PORT": "${USER_PROFILE_PORT}"
```
Comment (jerrytrieu) [inline deploy/service.json]: Need `USE_USER_PROFILE=true`

File: deploy/service.json [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+    "MESSAGING_MODE": "kafka",
+    "METRICS_DB": "${METRICS_DB}",
+    "PACKAGES_VERSION_FORMAT": ".+",
+    "SERVICE_NAME": "${CORE_JOB_NAME}",
+    "SERVICE_SCOPE": "internal",
+    "SERVICE_TYPE": "http",
+    "SERVICE_IMAGE": "advancedtelematic/sota-core:${DOCKER_TAG}",
+    "SERVICE_HEALTH_CHECK": "curl --show-error --silent %<host>s:%<port>s/health",
+    "TUF_REPOSERVER_HOST": "${TUF_REPOSERVER_HOST}",
+    "TUF_REPOSERVER_PORT": "${TUF_REPOSERVER_PORT}",
```
Comment (jerrytrieu) [inline deploy/service.json]: Need `USE_TUF=true`

Comment (jerrytrieu) [inline deploy/service.json]: In the `-pub` too

Comment (alexhumphreys) [inline deploy/service.json]: cool, updated.

File: deploy/service.json [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+    "INFLUXDB_URI": "${INFLUXDB_URI}",
+    "JWT_SECRET": "${JWT_SECRET}",
+    "JAVA_OPTS": "-Xmx800m",
+    "KAFKA_BOOTSTRAP_SERVERS": "${KAFKA_BOOTSTRAP_SERVERS}",
+    "KAFKA_HOST": "${KAFKA_HOST}",
+    "KAFKA_TOPIC_SUFFIX": "${KAFKA_TOPIC_SUFFIX}",
+    "MESSAGING_MODE": "kafka",
+    "METRICS_DB": "${METRICS_DB}",
+    "PACKAGES_VERSION_FORMAT": ".+",
+    "SERVICE_NAME": "${CORE_JOB_NAME}",
```
Comment (Danten) [inline deploy/service.json]: Should this be
```
"SERVICE_NAME": "${CORE_JOB_NAME}-pub",
```

Comment (alexhumphreys) [inline deploy/service.json]: Yes it should. Updated.

**Pull request merged**